integrations/kosli/manifest.json

```diff
@@ -38,10 +38,14 @@
         "method": "POST",
         "headers": [
           {
-            "name": "x-ld-signature",
-            "value": "{{secret}}"
+            "name": "Content-Type",
+            "value": "application/json"
           }
-        ]
+        ],
+        "hmacSignature": {
+          "headerName": "X-LD-Signature",
+          "hmacSecretFormVariableKey": "secret"
```

Comment:
The `{{secret}}` syntax in the existing header is Handlebars syntax in order to use the value of the secret formVariable defined on line 27. 

If the plan is to only refer to the key and process the formVariable value elsewhere, could you please provide more detail on how you are planning to achieve that?

Reply:
i told @InTheCloudDan i think we do it this way! i looked at the other integrations using hmac config via manifest (`compass` and `msteams`) and i think counterintuitively we actually want the field name in this field as opposed to the field value

Reply:
Makes sense, just realised we're switching over to the inbuilt HMAC now

Reply:
@monsagri we indeed copied what we saw compass do in their integration

Is there anything else you need on our end?